src/color/alert.json

```diff
@@ -2,152 +2,53 @@
   "color": {
     "alert": {
       "notification": {
-        "onLight": {
-          "value": "{color.brand.atlas.400.value}",
-          "public": false,
-          "onLight": true,
-          "usage": "Notification color on light backgrounds",
-          "wcag": "AAA",
-          "deprecated": true,
-          "comment": "Deprecated token",
-          "reference": "color-alert-notification-default",
-          "version": "3.15.0"
-        },
         "default": {
           "value": "{color.brand.atlas.400.value}",
           "public": true,
           "default": true,
           "usage": "Notification color on light backgrounds",
           "wcag": "AAA",
-          "deprecated": false
-        },
-        "onDark": {
-          "value": "{color.brand.breeze.300.value}",
-          "public": false,
-          "onDark": true,
-          "usage": "Notification color on dark backgrounds",
-          "wcag": "AAA",
-          "deprecated": true,
-          "comment": "Deprecated token",
-          "reference": "n/a",
-          "version": "3.12.0"
+          "deprecated": true
```

Comment:
None of the newly deprecated tokens have the additional data required to fill out the 'deprecated tokens' page in the doc site. We should have the following added to each deprecated token. 

`reference` - this is a reference to the new token to be used of there is one designated 
`version` - this is used to communicate with wich release these tokens were listed as deprecated